### ISSUE TITLE
Incorrect severity level

### ISSUE THREAD

**Issue by zypA13510** [AtomLinter/linter-xmllint#99]
When running `xmllint myXML.xml`, the result is:
```
myXML.xml:4: namespace warning : xmlns: URI my.short.namespace is not absolute
```
This warning can be suppressed when running `xmllint --nowarning myXML.xml`.

However, this warning is displayed as severity: Error inside the linter panel, and marked in the XML file as such. AFAIK we do have severity info/warning/error from the linter package API, so we should mark it according to the output from xmllint.

I'm not sure if all xmllint output follows the same format (just started using it). If it is, this can be easily fixed, I think?
